hack/test-go.sh

```diff
@@ -209,7 +208,7 @@ reportCoverageToCoveralls() {
 IFS=',' read -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
 for apiVersion in "${apiVersions[@]}"; do
   echo "Running tests for APIVersion: $apiVersion"
-  runTestsForVersion $apiVersion "${@}"
+  runTestsForVersion "$apiVersion" "${@}"
```

Comment:
Style guide suggests using braces around most variable names *except* special variables like `$@` or `$1` etc.
https://google-styleguide.googlecode.com/svn/trunk/shell.xml?showone=Variable_expansion#Variable_expansion

So this would be slightly better:

```shell
  runTestsForVersion "${apiVersion}" "$@"
```